libr/core/cmd_anal.c

```diff
@@ -871,6 +871,7 @@ static bool cmd_anal_aaft(RCore *core) {
 	r_reg_arena_push (core->anal->reg);
 	// Iterating Reverse so that we get function in top-bottom call order
 	r_list_foreach_prev (core->anal->fcns, it, fcn) {
+		r_reg_arena_zero (core->anal->reg);
 		r_core_cmd0 (core, "aei");
```

Comment:
actually i dont see the point on running aei and aeim on every function. we can do that once before the loop. so my pseudocode will look like this:

push,aei,zero,aeim ( aaft loop ) pop